src/core/java/com/enderio/core/client/item/ItemTooltip.java

```diff
@@ -0,0 +1,30 @@
+package com.enderio.core.client.item;
+
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.fluids.FluidStack;
+
+
+import java.util.List;
+
+public class ItemTooltip {
+
+    public static void addShiftKeyMessage(List<Component> source){
+        //TODO: add translation
+        if(!Screen.hasShiftDown()){
+            source.add(Component.literal("<Hold shift>"));
+        }
+
+    }
+
+    public static void addFluidTankMessage(List<Component> source, FluidStack stack, int capacity){
```

Comment:
As a part of my prior comment, this would then be performed inside of the item itself.